xbmc/filesystem/FileCache.h

```diff
@@ -32,11 +32,11 @@ namespace XFILE
   class CFileCache : public IFile, public CThread
   {
   public:
-    CFileCache(bool useDoubleCache=false);
-    CFileCache(CCacheStrategy *pCache, bool bDeleteCache=true);
+    CFileCache(const unsigned int flags);
```

Comment:
I always do this regardless, as a precaution. I just want to make sure I don't modify any of the arguments passed.